README.md

```diff
@@ -26,8 +26,10 @@ and resource utilization.
 
 What's more, DLRover provides extension libraries of PyTorch and TensorFlow to Speed Up Training.
 
-- [ATorch](atorch/README.md): an extension library of PyTorch to Speed Up Training of Large LLM.
-- [TFPlus](tfplus/README.md): an extension library of TensorFlow to Speed Up Training of Search, Recommendation and Advertisement.
+- [ATorch](https://github.com/intelligent-machine-learning/atorch): an extension library of PyTorch to Speed Up Training of Large LLM.
+- [TFPlus](https://github.com/intelligent-machine-learning/tfplus): an extension library of TensorFlow to Speed Up Training of Search, Recommendation and Advertisement.
+
+UPDATE 20241127: ATorch and TFPlus have moved to the independent repositories and import as git submodules. so you shoud run `git submodule update --init --recursive` to get the latest code before you build the project.
```

Comment:
ATorch and TFPlus are independent python packages, and DLRover package build does not need these tow modules. Thus, it is not necessary to add them as submodules.

Reply:
Yep, +1

I recommend directly removing this part of the content.

Reply:
OK, I updated the documents and removed the submodules by the following commits.